### PR TITLE
Add Diminishing_None to Feral Charge. And Add Blizzlike Animations to Flags / Chest Captures.

### PR DESCRIPTION
**Pull request by lineagedr**
Closes https://github.com/azerothcore/azerothcore-wotlk/issues/648

Credit: AnonXS @L4G_Core

**Changes proposed:**
-
Add DIMINISHING_NONE to Feral Charge.
-  
Add Blizzlike Animations to Chest / Flag Captures:
https://www.youtube.com/watch?v=XWDrwvqUoKQ&feature=youtu.be&t=35s
-  
Added more missing animations like: Demonic Circle: Summon, Ritual of Souls, Ritual of Summoning
https://www.youtube.com/watch?v=nVIz2flnmX8&feature=youtu.be
-  

**Target branch(es):** 1.x/2.x etc.
3.3.5a
**Issues addressed:** Closes #
https://github.com/azerothcore/azerothcore-wotlk/issues/648
**Tests performed:** (Does it build, tested in-game, etc)
Tested and working.
**Known issues and TODO list:**

- [ ] 
- [ ] 

**NOTE** You no longer need to squash your commits, on merge we will squash it for you. (GitHub added a new feature)

### CODE REVIEW

File: src/game/Spells/SpellMgr.cpp [hunk 1]
```diff
@@ -132,6 +132,9 @@ DiminishingGroup GetDiminishingReturnsGroupForSpell(SpellInfo const* spellproto,
             // Faerie Fire
             else if (spellproto->SpellFamilyFlags[0] & 0x400)
                 return DIMINISHING_LIMITONLY;
+            // Feral Charge Root Effect
+            else if (spellproto->Id == 45334)
```
Comment (ShinDarth) [inline src/game/Spells/SpellMgr.cpp]: it would be better to wrap those numbers into costants

**Comment by lineagedr** [comment]
oops that's not what I wanted to do..

File: src/game/Spells/SpellMgr.cpp (hunk 1 above)
Comment (BarbzYHOOL) [reply src/game/Spells/SpellMgr.cpp]: Give an example because if someone is porting from TC, it might not be obvious.

**Comment by lineagedr** [comment]
@BarbzYHOOL Yes, I was the guy with the sunwell i80 repack.

**Pull request merged**